apps/src/templates/instructions/TopInstructionsCSF.story.jsx

```diff
@@ -122,6 +124,10 @@ const STORIES = {
   'Markdown instructions': {
     longInstructions: true,
   },
+  'Only long instructions, no short': {
+    longInstructions: true,
+    disableShortInstructions: true
+  },
```

Comment:
Hmm, how is this different from [storybook -> TopInstructionsCSF -> Minimal Example](https://code-dot-org.github.io/cdo-styleguide/?selectedKind=TopInstructionsCSF&selectedStory=Minimal%20Example)?  Was that entry throwing a PropTypes error before, and now it's not?

Reply:
That one has short instructions but no long ones, this is the opposite

Reply:
Ah, I see 👍